test/index.html

```diff
@@ -16,7 +16,8 @@
   <script>
     WCT.loadSuites([
       'basic.html',
-      'basic.html?dom=shadow'
+      'basic.html?dom=shadow',
+      'basic.html?dom=shadow&lazyRegister=true&useNativeCSSProperties=true'
```

Comment:
I really only care about `useNativeCSSProperties` here, but the other two are necessary to enable that flag. (link in comment above) :shrug: